test_groups/signac_project_document.json

```diff
@@ -0,0 +1 @@
+{"_status": {}}
```

Comment:
The `test_groups` directory should probably be a sub-directory of `tests/`.

Reply:
I planned on ditching it once we were ready to merge. It was just a testing ground.

Reply:
Can you remove it *now* and just keep it locally to reduce the noise on this PR?